code_folder/22_glue_s3_athena_pipeline.py

```
@@ -0,0 +1,617 @@
+    """Export query results to other data stores."""
+
+    @staticmethod
+    def to_rds_postgres(df: pd.DataFrame, table_name: str, connection_string: str):
+        """Export DataFrame to RDS PostgreSQL."""
+        from sqlalchemy import create_engine
+        engine = create_engine(connection_string)
+        df.to_sql(table_name, engine, if_exists="replace", index=False, method="multi")
+        print(f"[Export] Wrote {len(df)} rows to RDS table '{table_name}'")
+
+    @staticmethod
+    def to_dynamodb(df: pd.DataFrame, table_name: str, key_column: str):
```

Ruff (0.14.14)

[warning] Unused static method argument: `key_column`

(ARG004)